acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,32 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
+  tag 'risk:high'
 
-agents.each do |agent|
-  on agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby"
-  ruby_path = stdout.chomp
+  agents.each do |agent|
+    ruby_path = on(agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby").stdout.chomp
```

Comment:
Example of use (I just learned about this yesterday myself): https://github.com/puppetlabs/puppet/pull/5827/files#diff-c2d6fd4eb072d50a0c31158ba3ed4f67